tests/Doctrine/Tests/DBAL/Functional/Schema/MySqlSchemaManagerTest.php

```diff
@@ -280,12 +278,12 @@ public function testColumnCharsetChange(): void
     public function testColumnCollation(): void
     {
         $table = new Table('test_collation');
-        $table->addOption('collate', $collation = 'latin1_swedish_ci');
+        $table->addOption('collate', 'latin1_swedish_ci');
         $table->addOption('charset', 'latin1');
         $table->addColumn('id', 'integer');
         $table->addColumn('text', 'text');
         $table->addColumn('foo', 'text')->setPlatformOption('collation', 'latin1_swedish_ci');
-        $table->addColumn('bar', 'text')->setPlatformOption('collation', 'utf8_general_ci');
+        $table->addColumn('bar', 'text')->setPlatformOption('collation', 'utf8mb4_general_ci');
```

Comment:
From the [documentation](https://mariadb.com/kb/en/unicode/):

> [MariaDB 10.6](https://mariadb.com/kb/en/what-is-mariadb-106/), `utf8` is an alias for `utf8mb3`.

In order to test schema introspection, we should use a charset that isn't an alias to be introspected as is.